### PR TITLE
Added totals for print_stats (command line output)

### PR DESCRIPTION
**Pull request by SecretObsession**
Updated the output after locust is stopped to include the totals for the columns which were missing values. (Avg, Min, Max, Median.)

### CODE REVIEW

File: locust/stats.py [hunk 1]
```diff
@@ -454,12 +463,33 @@ def print_stats(stats):
...
+
+        if total_max < r.max_response_time:
+            total_max = r.max_response_time
+
+        if total_min > r.min_response_time:
+            total_min = r.min_response_time
+
         console_logger.info(r)
+
+    total_average = float(running_avg / total_stat_methods)
```
Comment (melissa-kam) [inline locust/stats.py]: If locust is exited before executing any requests, this line will throw a ZeroDivisionError.

File: locust/stats.py [hunk 2]
```diff
@@ -467,7 +497,7 @@ def print_stats(stats):
     except ZeroDivisionError:
         fail_percent = 0
 
-    console_logger.info((" %-" + str(STATS_NAME_WIDTH) + "s %7d %12s %42.2f") % ('Total', total_reqs, "%d(%.2f%%)" % (total_failures, fail_percent), total_rps))
+    console_logger.info((" %-" + str(STATS_NAME_WIDTH) + "s %7d %12s %7d %7d %7d %10d %7.2f") % ('Total', total_reqs, "%d(%.2f%%)" % (total_failures, fail_percent), total_average, total_min, total_max, total_median, total_rps))
```
Comment (melissa-kam) [inline locust/stats.py]: Again, in the case of exiting before executing, total_min will be None and this line will fail with a TypeError.

File: locust/stats.py [hunk 3]
```diff
@@ -407,6 +408,14 @@ def median_from_dict(total, count):
         pos -= count[k]
 
 
+def median(input_list):
+    input_list.sort()
+    if len(input_list) % 2 == 1:
```
Comment (melissa-kam) [inline locust/stats.py]: When the length of input_list is 0, this will fail with an IndexError.

**Comment by SecretObsession** [comment]
@melissa-kam Thanks for the feedback Melissa. I've made the recommended changes to the PR.

**Comment by bsiver** [comment]
Any update on this?  I have a need to run Locust headless, and would like to see the total stats displayed in the console output.  Thanks!

**Comment by mbeacom** [comment]
@SecretObsession Would you be interested in having this merged? Would you mind rebasing the branch for review?

Thank you!

**Comment by cyberw** [comment]
Hi @SecretObsession ! There has been no activity in this PR for a long time and it has conflicts so I'm closing this. Please let me know if you want to reopen!

**Pull request closed without merging**